common/aggregation_scheme.py

```diff
@@ -0,0 +1,44 @@
+from utils import constants
+import numpy as np
+
+DEFAULT_AGGREGATION_SCHEME = constants.AGGREGATITON_SCHEME_AVERAGE
+
+def get_aggregation_scheme(server_model, CLIENT_IDS, CLIENT_NETWORKS):
+    if DEFAULT_AGGREGATION_SCHEME == constants.AGGREGATITON_SCHEME_AVERAGE:
+        return AggregationAverage(server_model, CLIENT_IDS, CLIENT_NETWORKS)
+
+
+class AggregationAverage():
+
+    def __init__(self, server_model, CLIENT_IDS, CLIENT_NETWORKS):
+        self.server_model = server_model
+        self.client_ids = CLIENT_IDS
+        self.client_networks = CLIENT_NETWORKS  
+
+    def __perform_average(self, weight_dimension, bias_dimension):
```

Comment:
why are weight_dimension and bias_dimension parameters if you can derive them from class variables?

can't you calculate this in this method?